src/assets/i18n/pt-BR.json5

```diff
@@ -1737,77 +1504,54 @@
-  // TODO New key - Add a translation
-  "collection.source.controls.import.running": "Importing...",
+  "collection.source.controls.import.running": "Importando...",
   // "collection.source.controls.import.failed": "An error occurred during the import",
-  // TODO New key - Add a translation
-  "collection.source.controls.import.failed": "An error occurred during the import",
+  "collection.source.controls.import.failed": "Um erro ocorreu durante a importação",
   // "collection.source.controls.import.completed": "The import completed",
   // TODO New key - Add a translation
   "collection.source.controls.import.completed": "The import completed",
   // "collection.source.controls.reset.submit.success": "The reset and reimport has been successfully initiated",
   // TODO New key - Add a translation
   "collection.source.controls.reset.submit.success": "The reset and reimport has been successfully initiated",
   // "collection.source.controls.reset.submit.error": "Something went wrong with initiating the reset and reimport",
-  // TODO New key - Add a translation
-  "collection.source.controls.reset.submit.error": "Something went wrong with initiating the reset and reimport",
+  "collection.source.controls.reset.submit.error": "Alguma coisa errada durante a inicialização da redefinição e reimportação",
   // "collection.source.controls.reset.failed": "An error occurred during the reset and reimport",
-  // TODO New key - Add a translation
-  "collection.source.controls.reset.failed": "An error occurred during the reset and reimport",
+  "collection.source.controls.reset.failed": "Um erro ocorreru durante o reset e reimportação",
   // "collection.source.controls.reset.completed": "The reset and reimport completed",
-  // TODO New key - Add a translation
-  "collection.source.controls.reset.completed": "The reset and reimport completed",
+  "collection.source.controls.reset.completed": "Completou o reset e a reimportação",
   // "collection.source.controls.reset.submit": "Reset and reimport",
-  // TODO New key - Add a translation
-  "collection.source.controls.reset.submit": "Reset and reimport",
+  "collection.source.controls.reset.submit": "Resetar e reimportar",
   // "collection.source.controls.reset.running": "Resetting and reimporting...",
-  // TODO New key - Add a translation
-  "collection.source.controls.reset.running": "Resetting and reimporting...",
+  "collection.source.controls.reset.running": "Resetando e importando...",
   // "collection.source.controls.harvest.status": "Harvest status:",
-  // TODO New key - Add a translation
-  "collection.source.controls.harvest.status": "Harvest status:",
+  "collection.source.controls.harvest.status": "Status do Harvest:",
   // "collection.source.controls.harvest.start": "Harvest start time:",
-  // TODO New key - Add a translation
-  "collection.source.controls.harvest.start": "Harvest start time:",
+  "collection.source.controls.harvest.start": "Hora de ínicio Harvest:",
```

Comment:
For `Harvest` you can use `Colecta`

Reply:
Is not usual translate Harvest in Brazilian Portuguese